slides/k8s/concepts-k8s.md

```diff
@@ -231,25 +241,21 @@ Yes!
 
 ---
 
-## Kubernetes resources
+## Interacting with Kubernetes
 
-- The Kubernetes API defines a lot of objects called *resources*
+- We will interact with our Kubernetes cluster through the Kubernetes API
 
-- These resources are organized by type, or `Kind` (in the API)
+- The Kubernetes API is (mostly) RESTful
+
+- It allows us to create, read, update, delete *resources*
 
 - A few common resource types are:
 
   - node (a machine — physical or virtual — in our cluster)
-  - pod (group of containers running together on a node)
-  - service (stable network endpoint to connect to one or multiple containers)
-  - namespace (more-or-less isolated group of things)
-  - secret (bundle of sensitive data to be passed to a container)
```

Comment:
Are we going to skip namespaces and secrets in this bit now?

Reply:
Yes; I found that this selection was arbitrary (why talk about these, and not others?) and I initially wanted to be as concise as possible. And when I delivered it, I realized it worked just as well.

Reply:
Cool!